src/Comet/Styles/ProgressBarStyle.cs

```diff
@@ -5,7 +5,6 @@ namespace Comet.Styles
 {
 	public class ProgressBarStyle : ViewStyle
 	{
-		public StyleAwareValue<ControlState, Color> TrackColor { get; set; }
```

Comment:
Why is this being deleted?

Reply:
MAUI API changed, there's no TrackColor property in Progress class